og_ui/og_ui.module

```diff
@@ -75,18 +76,20 @@ function og_ui_entity_type_save(EntityInterface $entity) {
   }
 
   // Change the field target type and bundle.
+  if ($field_storage = FieldStorageConfig::loadByName($entity_type_id, OgGroupAudienceHelper::DEFAULT_FIELD)) {
+    $target_type = $field_storage->getSetting('target_type');
+    if ($entity->og_target_type !== $target_type) {
```

Comment:
ok, couldn't hold myself :)

where in the UI to you see the posibility to change the target type?